pkg/check/gc/reserve.go

```diff
@@ -140,19 +145,19 @@ func CheckReserve(c *bee.Cluster, o Options) error {
 	}
 
 	// STEP 3: Upload chunks with high value batch, then create a low value batch, and confirm no chunks were garbage collected
```

Comment:
before this step, I would add an explicit check that the node `Has(pinnedChunk.Address()`

Reply:
Done

Reply:
I meant to add a check that the node indeed stores the chunk and that it has not been GCd:
```go
has, err := client.HasChunk(ctx, pinnedChunk.Address())
.....
if !has { return error("expected node pin uploaded chunk") }
```

Reply:
Sorry, my bad, fixed.